src/aics-image-viewer/components/App/index.tsx

```diff
@@ -103,6 +104,8 @@ const defaultProps: AppProps = {
     showBoundingBox: false,
     view: "3D", // "XY", "XZ", "YZ"
     mode: "default", // "pathtrace", "maxprojection"
+    backgroundColor: [0, 0, 0],
+    boundingBoxColor: [255, 255, 255],
```

Comment:
These seem like they should be in constants so you can have a default you use other places